stacks.go

```
@@ -29,10 +44,39 @@ func (c *Client) StacksList(ctx context.Context) ([]Stack, error) {
 		Endpoint: "/features/stacks",
 	}
 
-	resmap := map[string][]Stack{}
+	resmap := map[string][]jsonStack{}
 	err := c.ScalingoAPI().DoRequest(ctx, req, &resmap)
 	if err != nil {
 		return nil, errgo.Notef(err, "fail to request Scalingo API")
 	}
-	return resmap["stacks"], nil
+
+	return jsonStackToStack(resmap["stacks"]), nil
+}
+
+func jsonStackToStack(s []jsonStack) []Stack {
+	var stacks []Stack
+
+	for _, stack := range s {
+		deprecationDate, _ := time.Parse("2006-01-02", stack.DeprecatedAt)
```

You shoudn't ignore the error btw